### PR TITLE
The first version of Pi-Adapter for running NTOU KWGIUH model from De…

### PR DESCRIPTION
**Pull request by JackChao1111**
…lft-FEWS.

Signed-off-by: JackChao1111 <bochao.liang1111@gmail.com>

### CODE REVIEW

File: PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/KWGIUHPostAdapter.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+			TimeSeriesArray timeSeriesArray = TimeSeriesLightUtils.readPiTimeSeries( rainfallPath ).get( 0 );
+
+			logger.log( LogLevel.INFO, "KWGIUHPostAdapter: Filling timeseries header for model output." );
+			SimpleTimeSeriesContentHandler handler = new SimpleTimeSeriesContentHandler();
+			TimeSeriesLightUtils.fillPiTimeSeriesHeader( handler, timeSeriesArray.getHeader().getLocationId(),
+					"Q_simulated", "m^3/s" );
+			DateTime startTime = new DateTime( timeSeriesArray.getStartTime() );
+
+			logger.log( LogLevel.INFO, "KWGIUHPostAdapter: Reading model output and convert to Pi-XML format." );
+			List<String> lines = Files.readAllLines( modelOutputPath, Charset.forName( "Cp1252" ) );
```
Comment (Vipcube) [inline PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/KWGIUHPostAdapter.java]: ChartSet not UTF-8?

File: PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/argument/ExecutableArguments.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import tw.fondus.fews.adapter.pi.argument.PiIOArguments;
+/**
+ * Model executable-adapter and post-adapter arguments for running NTOU KWGIUH model.
+ * 
+ * @author Chao
+ *
+ */
+
```
Comment (Vipcube) [inline PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/argument/ExecutableArguments.java]: Remove space.

File: PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/argument/PreAdapterArguments.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import lombok.EqualsAndHashCode;
+import tw.fondus.fews.adapter.pi.argument.PiIOArguments;
+
+/**
+ * Model pre-adapter arguments for running NTOU KWGIUH model.
+ * 
+ * @author Chao
+ *
+ */
+
```
Comment (Vipcube) [inline PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/argument/PreAdapterArguments.java]: Remove space.

File: PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/KWGIUHPostAdapter.java (hunk 1 above)
Comment (JackChao1111) [reply PI-Adapter-NTOU-KWGIUH/src/main/java/tw/fondus/fews/adapter/pi/kwgiuh/ntou/KWGIUHPostAdapter.java]: ANSI

**Pull request merged**